studiopress/atomic-blocks
# Add pricing table section design

**Summary of change:**
This PR adds the pricing section design to the layout modal.

**This PR has been:**

**How to test:**
Add the layout block, select the Pricing Table section and visually test it.

**See:** #186 

**Suggested Changelog Entry:**
Add Pricing Table section design.
